packages/combo-box/src/vaadin-combo-box.d.ts

```diff
@@ -62,77 +62,60 @@ export interface ComboBoxEventMap<TItem> extends HTMLElementEventMap {
 }
 
 /**
- * `<vaadin-combo-box>` is a combo box element combining a dropdown list with an
- * input field for filtering the list of items. If you want to replace the default
- * input field with a custom implementation, you should use the
- * [`<vaadin-combo-box-light>`](#/elements/vaadin-combo-box-light) element.
- *
- * Items in the dropdown list must be provided as a list of `String` values.
- * Defining the items is done using the `items` property, which can be assigned
- * with data-binding, using an attribute or directly with the JavaScript property.
+ * `<vaadin-combo-box>` is a web component for choosing a value from a filterable list of options
+ * presented in a dropdown overlay. The options can be provided as a list of strings or objects
+ * by setting [`items`](#/elements/vaadin-combo-box#property-items) property on the element.
  *
  * ```html
- * <vaadin-combo-box
- *     label="Fruit"
- *     items="[[data]]">
- * </vaadin-combo-box>
+ * <vaadin-combo-box id="combo-box"></vaadin-combo-box>
  * ```
  *
  * ```js
- * combobox.items = ['apple', 'orange', 'banana'];
+ * document.querySelector('#combo-box').items = ['apple', 'orange', 'banana'];
  * ```
  *
  * When the selected `value` is changed, a `value-changed` event is triggered.
  *
  * ### Item rendering
  *
- * `<vaadin-combo-box>` supports using custom renderer callback function for defining the
- * content of `<vaadin-combo-box-item>`.
+ * To customize the content of the `<vaadin-combo-box-item>` elements placed in the dropdown, use
+ * [`renderer`](#/elements/vaadin-combo-box#property-renderer) property which accepts a function.
+ * The renderer function provides `root`, `comboBox`, and `model` arguments.
```

Comment:
"The renderer function takes `root`, `comboBox`, and `model` as parameters"?
"The renderer function is executed with `root`, `comboBox`, and `model` as arguments"?

Reply:
Thanks, I tend to misuse those 🙈

Reply:
Done.